electron/wine/runtimes/runtimes.ts

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios'
+import { existsSync, mkdirSync, unlinkSync, writeFileSync } from 'graceful-fs'
+import { join } from 'path'
+import { runtimePath } from './../../constants'
+import { logDebug, LogPrefix } from './../../logger/logger'
+import { Runtime, RuntimeName } from './../../types'
+import { extractTarFile, getAssetDataFromDownload } from './util'
+
+async function _get(): Promise<Runtime[]> {
+  mkdirSync(runtimePath, { recursive: true })
```

Comment:
Please catch possible errors here and log them!

Reply:
We're using mkdirSync without error handling in [a](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/main/electron/config.ts#L220) [lot](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/main/electron/launcher.ts#L353) [of](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/main/electron/main.ts#L1177) [other](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/main/electron/main.ts#L1177) [places](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/main/electron/main.ts#L1177) as well